docs/users.md

```diff
@@ -64,28 +70,26 @@ By default, each newly uploaded dataset can only be accessed by `Admins` and `Da
 | Create scripts (visible to everyone)            	| Yes   	| No              	| Yes          	| No          	|
 | Upload Datasets via UI                           	| Yes   	| Yes             	| Yes          	| No          	|
 | Set team access permissions upon dataset upload    	| Yes   	| Yes              	| Yes          	| No          	|
-| Get tasks again after cancelling an instance     	| Yes   	| No              	| Yes          	| No          	|
+| Get tasks again after canceling an instance     	| Yes   	| No              	| Yes          	| No          	|
 | Access to wK Statistics Menu  	| Yes   	| No              	| Yes          	| No          	|
 
 
-## Registering New Users
-
-New users can signup for webKnossos by clicking on the `Register Now` button in the navigation bar.
-In the registration form, they need to specify some basic information, and confirm acceptance of the privacy agreement.
-
-If there is just one organization in the webKnossos instance, they are assigned to that organization automatically but need to be activated by an administrator, compare below.
+## Sign Up & Invitations for New User Accounts
+On webKnossos.org, users can either sign up for a webKnossos account by themselves or be invited through email to join an existing organization by co-workers or collaborators. On private webKnossos instances, users can only join through an email invitation.
 
 !!!info
-    You can also invite users to your webKnossos organization, by clicking the `Invite Users` button at the top of the user list. This will open a popup where you can enter a list of email addresses, which will receive a custom invitation link. Users that click on this link are automatically assigned to your organization, and will not need manual activation.
+    As an admin or team manager, you can invite users to join your webKnossos organization by clicking the `Invite Users` button at the top of the `Users` list available from the `Admin` menu in the navbar. This will open a popup where you can enter a list of email addresses, which will receive a custom invitation link. Users that click on this link are automatically assigned to your organization, and will not need manual activation.
 
     ![Send an invite link to new users](./images/users_invite.png)
 
 
 ## Activating New Users
```

Comment:
Maybe get rid of that section entirely?

Reply:
this is still relevant for self-hosted instances